docs/guides/testing.md

```diff
@@ -146,6 +147,9 @@ Tagging an example group (`describe` or `context`) extends to all of the contain
 
 Multiple tags can be specified by giving an [`Enumerable`](https://crystal-lang.org/api/latest/Enumerable.html), such as [`Array`](https://crystal-lang.org/api/latest/Array.html) or [`Set`](https://crystal-lang.org/api/latest/Set.html).
 
+Be sure to use `--tag="my-tag"` syntax when there is a directory name that matches your tag name in your project or else they will be parsed as directories instead of tag filters.  For example, if you have some tests that run in a Terraform environment and you have put all of your terraform scripts in `terraform/example.tf` and have also tagged some tests as `tags: "terraform"`
```

Comment:
I don't think "Terraform" has enough relevance here to be used without any introduction

Reply:
thanks for the review.

I often find concrete examples useful in docs when dealing with corner cases, but i can definitely clean it up when I'm back from vacation. 

Unless it is decided this behavior is actually a crystal bug we should fix.